capture.sh

```diff
@@ -11,9 +20,28 @@ else
   exit 1
 fi
 
+PERIODI=4 #default to 4 seconds between captures. Can be changed with --period command line option
```

Comment:
Why did you decide to call this variable `PERIODI` instead of `PERIOD`?

Reply:
`PERIOD` indeed is a better variable name. I initially changed the script without the intent to start a PL. I had a small issue with `PERIOD`, so I changed it to `PERIODI` and forgot about it. It's `PERIOD` again without any problems now.